pma/README.md

```diff
@@ -0,0 +1,12 @@
+# PHPMyAdmin
+
+Easily install PHPMyAdmin for your Docksal project. Make it easy to access the database and examine directly rather then through the CLI.
+
+```bash
+fin addon install pma
+```
+
+## Usage once installed
+
+- `fin pma disable` to disable
+- `fin pma enable` to re-enable
```

Comment:
Could you please append information how to access PMA after you have installed the addon?